app/src/main/java/com/github/scribeWizTeam/scribewiz/util/Export.kt

```diff
@@ -0,0 +1,66 @@
+import android.content.Context
+import android.content.Intent
+import android.util.Log
+import androidx.core.content.FileProvider
+import java.io.File
+
+class Export {
```

Comment:
Consider renaming to be more specific

Reply:
You're right it could have a more suitable name.